passkeys-backend/.env.example

```diff
@@ -0,0 +1,4 @@
+API_URL=
+RELYING_PARTY=
+ACCOUNT_SID=
+AUTH_TOKEN=
```

Comment:
Delete ACCOUNT_SID and AUTH_TOKEN since those are automatically created for all function templates. 

(you can test it with https://github.com/twilio-labs/function-templates/tree/main/verify if you're curious how it looks)

Reply:
I did this using the getTwilioClient() function and adding the comment inside the readme about enabling this feature in the twilio console just like in the verify template,  however, I did need to add ACCOUNT_SID and AUTH_TOKEN in the enviroments variable in order to running it local with the twilio cli